csrc/ir/utils.cpp

```diff
@@ -1265,158 +1265,3 @@ int64_t getOperationCount(Val* val) {
 }
 
 } // namespace nvfuser::ir_utils
-
-namespace nvfuser::MmaOpUtils {
```

Comment:
This was only needed for defining `MmaOp::mAxes()` and friends, but:
1. Those methods are never used so I removed them and
2. We can reconstruct that information easily using `mma->axisMapping()`.